polaris/mesh/spherical.py

```diff
@@ -28,22 +28,22 @@ class SphericalBaseStep(Step):
     opts : jigsawpy.jigsaw_jig_t
         JIGSAW options for creating the mesh
     """
-    def __init__(self, task, name, subdir):
+    def __init__(self, component, name, subdir):
         """
```

Comment:
No, the current design is that steps belong one component only.  You also cannot currently run tasks from different components in the same suite (or "custom" suite defined from a list of tasks), in part because the `-p` flag points to the directory where the model associated with the component exists.